support-lambdas/stripe-intent/cfn.yaml

```diff
@@ -17,6 +17,10 @@ Mappings:
       StripeConfig: test
     PROD:
       StripeConfig: live
+  Constants:
+    Alarm:
+      Process: Follow the process in https://docs.google.com/document/d/1_3El3cly9d7u_jPgTcRjLxmdG2e919zCLvmcFCLOYAk/edit
+      Urgent: URGENT 9-5 -
```

Comment:
it's not essential to do the join and the constants for the alarm name, this was just because we changed it a couple of times in one that had loads of alarms.  Feel free to just inline it to make it more intelligable

Reply:
oh just realised I was looking at the last commit only somehow, you didn't actually add this for the one alarm, ignore it!